spec/models/event_spec.rb

```diff
@@ -17,4 +17,25 @@
       expect(event).to be_valid
     end
   end
+
+  describe '#in_week' do
+    let(:start_time) { Time.now.beginning_of_week }
+    let!(:events) {
```

Comment:
how about dividing the let content into two:
`let!(:expected_events) { ... }`
`let!(:other_event) { ... }`